imagej/imagej-ops2/src/test/java/net/imagej/ops2/AbstractOpTest.java

```diff
@@ -75,20 +75,14 @@
  */
 public abstract class AbstractOpTest{
 
-	protected static OpEnvironment ops;
+	protected static OpEnvironment ops = new DefaultOpEnvironment();
```

Comment:
Maybe add `final` here, just in case?

Reply:
Done in 62aa49651fa3d0fb939ba9ca7f47040268da7d40. @hinerm Do you see any problem with doing this? I am not sure, but I could imagine that pushing the data structure construction earlier, to class initialization time rather than whenever JUnit calls the `BeforeClass` methods, could potentially alter behavior? Everything does still pass for me locally though.

Reply:
>  I am not sure, but I could imagine that pushing the data structure construction earlier, to class initialization time rather than whenever JUnit calls the BeforeClass methods, could potentially alter behavior?

That's a good point, but since it's only in the tests, we can change it later if we find that it's an issue, no?

Reply:
> since it's only in the tests, we can change it later if we find that it's an issue, no?

Definitely; that was my thinking as well.